bin/combine_results.py

```diff
@@ -79,14 +79,13 @@ def main():
             testcase_count += 1
             for failure in testcase.iter('failure'):
                 if args.set_rc:
-                    rc = 1
+                    rc += 1
```

Comment:
This seems like quite an odd thing to do, I don't know of other applications that use the return code to indicate a count.

Reply:
It was a hack, certainly. Should I just write a local python script instead of modifying this script?

Reply:
I can grep the output. Not sure why I didn't think of that.